cmd/gcp/main.go

```diff
@@ -157,9 +159,19 @@ func newGCPStorage(ctx context.Context, signer note.Signer) (*storage.CTStorage,
 		return nil, fmt.Errorf("failed to initialize GCP Tessera storage driver: %v", err)
 	}
 
+	var antispam tessera.Antispam
+	// Persistent antispam is currently experimental, so there's no terraform or documentation yet!
```

Comment:
nit: I think there might be terraform now (at least in Tessera), although docs may need updating :)